server/test/e2e/covenantless/e2e_test.go

```diff
@@ -141,6 +151,158 @@ func TestUnilateralExit(t *testing.T) {
 	require.NotZero(t, lockedBalance)
 }
 
+func TestUnilateralExitWithAnchorSpend(t *testing.T) {
+	ctx := context.Background()
+	sdkClient, grpcClient := setupArkSDK(t)
+	defer grpcClient.Close()
+
+	_, boardingAddress, err := sdkClient.Receive(ctx)
+	require.NoError(t, err)
+
+	_, err = utils.RunCommand("nigiri", "faucet", boardingAddress)
+	require.NoError(t, err)
+
+	time.Sleep(5 * time.Second)
+
+	roundId, err := sdkClient.Claim(ctx)
+	require.NoError(t, err)
+
+	err = utils.GenerateBlock()
+	require.NoError(t, err)
+
+	_, err = sdkClient.Claim(ctx)
```

Comment:
why a second claim?

Also, let's use `sdkClient.UnilateralRedeem` instead of redoing the same steps here